cmd/sriovdp/manager.go

```diff
@@ -46,6 +48,7 @@ Network controller subclasses. ref: https://pci-ids.ucw.cz/read/PD/02
 		07	Infiniband controller
 		08	Fabric controller
 		80	Network controller
+		12	Processing Accelerator
```

Comment:
I think this is actually wrong.  Number 12 is correct for the PCI device class for processing accelerators as specified at https://pci-ids.ucw.cz/read/PD/

However, it does not appear to be a valid option for a *network* controller subclass as described at the URL listed on line 40.